Sensus.Shared/DataStores/Local/LocalDataStore.cs

```diff
@@ -214,7 +214,9 @@ public int WriteDataToZipFile(string zipPath, CancellationToken cancellationToke
                 // ensure that zip file is closed.
                 try
                 {
-                    if (zipFile != null)
+#if __ANDROID__ || __IOS__
+                    if (zipFile != null)
+#endif
```

Comment:
zipFile is only created above if we're in an Android or IOS build. As such this code wouldn't build in Sensus.Tests.Local without the if around zipFile. I changed it a little so that it is maybe a little more clear what is going on.